lib/utils.js

```diff
@@ -156,39 +156,53 @@ const config = values => {
  */
 const format = body => {
   const data = {};
-  Object.keys(body).forEach(key => {
-    if ((key === 'sunset-date') || (key === 'sunset-time') || (key === 'sunrise-date') || (key === 'sunrise-time')) {
-      data[key] = {
-        'value': body[key],
-      };
+  const single = Object.keys(body).filter(input => {
+    if ((input === 'sunset-date') || (input === 'sunset-time') || (input === 'sunrise-date') || (input === 'sunrise-time')) {
```

Comment:
@ayush2k if this is in place, can it supersede [this section in workflows](https://github.com/punchcard-cms/punchcard/blob/master/lib/routes/workflows.js#L93)?

Meaning...can that section in workflows.js be removed?

```
          // clear extraneous values
          const remove = ['sunrise-date', 'sunrise-time', 'sunset-time', 'sunset-date', 'submit', 'action', 'comment-textarea'];
          remove.forEach(rmv => {
            delete data[rmv];
          });
```

note please that there are two extra items in that one: `action` & `comment-textarea`